tests/phpunit/tests/block-supports/layout.php

```diff
@@ -173,4 +173,306 @@ function test_outer_container_not_restored_for_aligned_image_block_with_themejso
 
 		$this->assertSame( $expected, wp_restore_image_outer_container( $block_content, $block ) );
 	}
+
+	/**
+	 * Generates the CSS corresponding to the provided layout.
+	 *
+	 * @ticket 56467
+	 *
+	 * @dataProvider data_wp_get_layout_style
+	 *
+	 * @covers ::wp_get_layout_style
+	 *
+	 * @param array  $args {
+	 *       Arguments for the test function.
+	 *
+	 *      @type string  $selector                      CSS selector.
+	 *      @type array   $layout                        Layout object. The one that is passed has already checked the existence of default block layout.
+	 *      @type boolean $has_block_gap_support         Whether the theme has support for the block gap.
```

Comment:
```suggestion
	 *      @type bool $has_block_gap_support         Whether the theme has support for the block gap.
```
Core uses `bool` rather than `boolean`. [See here](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#phpdoc-tags).

🔢 Applies to other instances in the PR.